mw-webapp/src/component/header/Header.module.scss

```diff
@@ -71,6 +71,20 @@ $iconSize: 32px;
   fill: var(--primaryIconColor);
 }
 
+.secondaryBurgerMenu {
+  stroke: var(--secondaryStrokeColor);
+}
+
+.buttonLogin {
```

Comment:
Is it possible to use Button type  SUPERSPECIALBEAUTIDUL BUTTON instead of new styles?

Reply:
fixed